core/include/cib.h

```diff
@@ -44,9 +46,9 @@ typedef struct {
  *
  * @param[out] cib      Buffer to initialize.
  *                      Must not be NULL.
- * @param[in]  size     Size of the buffer, must not exceed MAXINT/2.
+ * @param[in]  size     Size of the buffer, must not exceed 65,534.
```

Comment:
This size value fills a bit mask. 0 is treated specially as an empty CIB. So the maximum value is 2**15.